docs/roomHandlers/fishingRoom.md

```diff
@@ -35,5 +35,4 @@ stateDiagram-v2
 
   BrokenLine --> AtEntrance: stop
   BrokenLine --> AtEntrance: fix
-  BrokenLine --> [*]: leave
```

Comment:
I'm curious why this was removed. It's valid. Was it confusing or something?

Reply:
I want the player to choose to fix the fishing rod or stop.

In future versions of this game if the player choose not to fix the rod then leaves the room then comes back at a later point, the fishing rod will still be broken.